src/resources/user/controller.py

```diff
@@ -0,0 +1,23 @@
+from flask_restx import Resource
+from flask_jwt_extended import jwt_required
+
+from .service import UserService
+from .dto import UserDto
+
+api = UserDto.api
+data_resp = UserDto.data_resp
+
+
+@api.route("/<string:uuid>")
+class UserController(Resource):
+    @api.doc(
+        "Get a specific user",
+        responses={
+            200: ("User data successfully sent", data_resp),
+            404: "User not found!",
+        },
+    )
+    @jwt_required
+    def get(self, uuid):
+        """ Get a specific user's data by their uuid """
+        return UserService.get_user_data(uuid)
```

Comment:
Define which route is useful for user resource, a user can only access to his/her own informations ?

Even if at the moment we can't access much information about the user, in the future there may be some stats.